kolibri_instant_schools_plugin/assets/src/views/sign-in-page/index.vue

```diff
@@ -17,9 +18,10 @@
           <k-textbox
```

Comment:
decided to only swap out user-facing strings to "phone number" rather than swapping out all variable names - "username" is still what's being sent to the server, after all. Right?

Reply:
Yes, the smaller the diff, the better. As I mentioned in slack, it will be beneficial to avoid unnecessary refactors to minimize conflicts and make it easier to compare with mainline kolibri.

>  "username" is still what's being sent to the server, after all. Right?

This relates to work that @jonboiser is working on for retrieving the list of profiles from a phone number.